### ISSUE TITLE
GitHub Pages Site and HTTPS

### ISSUE THREAD

**Issue by markhoney** [nathan/phosphorus#493]
Hi,

I was hoping to embed the output from your github.io site for Phosphorus (http://phosphorus.github.io/) in a project I'm working on, but I've spotted that there's a redirect from HTTPS to HTTP in the app.html page and this is causing my iframe embed (in an HTTPS page) to fail. You can also see this occurring in your embed.html page if it's loaded via SSL:

https://phosphorus.github.io/embed.html

Is there any chance of either having this redirect removed from app.html, or adding a second app page that allows HTTPS? Even better, a separate embed.html as well for SSL would be great, or a single embed.html that automagically points to the correct file for HTTP/HTTPS.

Thanks!

mark.

**Comment by nathan** [comment]
I recall this breaking in HTTPS for some reason, but that may have changed since Scratch started supporting TLS.

**Comment by tjvr** [comment]
@nathan Are you thinking of #302?

**Comment by nathan** [comment]
> @nathan Are you thinking of #302?

No. A few of the resources loaded from Scratch's servers didn't like being loaded over HTTPS, even though Scratch purportedly supported HTTPS when I was testing it. Again, this may have changed, but I don't have time right now to test is as obsessively as I would need to in order to fix this.
